mne/viz/_brain/_brain.py

```diff
@@ -2893,6 +2930,8 @@ def scale_data_colormap(self, fmin, fmid, fmax, transparent,
         %(verbose_meth)s
         """
         # XXX fmid/transparent/center not used, this is a bug
+        if 'ctable' not in self._data:
+            return
```

Comment:
There is no colormap but someone calls `scale_data_colormap` -- same as above, seems like it should be an error to do this